components/forgerock.key.manager/src/main/java/org/wso2/forgerock/client/ForgerockJWTValidatoImpl.java

```diff
@@ -0,0 +1,26 @@
+package org.wso2.forgerock.client;
+
+import com.github.jknack.handlebars.internal.lang3.StringUtils;
+import com.nimbusds.jwt.JWTClaimsSet;
+import org.wso2.carbon.apimgt.api.APIManagementException;
+import org.wso2.carbon.apimgt.impl.dto.JWTValidationInfo;
+import org.wso2.carbon.apimgt.impl.jwt.JWTValidatorImpl;
+import org.wso2.carbon.apimgt.impl.jwt.SignedJWTInfo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ForgerockJWTValidatoImpl  extends JWTValidatorImpl {
```

Comment:
class comment ?

Reply:
Added in 0cbcfc879bd87c85a4c14d959c2d28653f7fd391